arm-network/2016-09-01/swagger/network.json

```diff
@@ -7900,6 +7898,43 @@
       },
       "description": "VpnClientParameters"
     },
+    "TunnelConnectionHealth": {
+      "properties": {
```

Comment:
@Nilambari - You should not have asked the .net sdk team to merge the PR in the .net sdk until the swagger spec PR is merged. This creates inconsistency in the spec and the generated code when changes are made to the spec while it is still in review.

After the readOnly changes are made and the PR is merged, please make sure to regenerate the .net sdk and send a new PR.